src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToHtml5DateTimeLocalTransformer.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+/**
+ * @author Franz Wilding <franz.wilding@me.com>
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+class DateTimeToHtml5DateTimeLocalTransformer extends BaseDateTimeTransformer
+{
+    const HTML5_FORMAT = "Y-m-d\TH:i:s";
```

Comment:
should use single quotes

Reply:
Fixed, I've also put '\\T' in the middle, as it looks more correct, it works with single `\` because `\T` is not a valid combo, but really for a literal `\` it should be `\\`